user_sync/connector/directory_ldap.py

```diff
@@ -289,6 +292,18 @@ def iter_users(self, users_filter, extended_attributes):
             elif last_attribute_name:
                 self.logger.warning('No country code attribute (%s) for user with dn: %s', last_attribute_name, dn)
 
+            uid_value = LDAPValueFormatter.get_attribute_value(record, six.text_type('uid'))
```

Comment:
Appears to be a remnant of the original design, which had the LDAP connector make an additional LDAP query to get direct membership groups. 

See https://github.com/adobe-apiplatform/user-sync.py/commit/73592db9ad1d9021d39a1d1ab5f794a03507ac6f

I don't think we need this anymore since we're now using `memberOf` to get direct groups.